tests/integration/modules/test_status.py

```diff
@@ -44,6 +44,7 @@ def test_status_saltmem(self):
         ret = self.run_function('status.saltmem')
         self.assertTrue(isinstance(ret, int))
 
+    @skipIf(salt.utils.is_darwin(), 'status.diskusage not currently supported on macosx')
```

Comment:
Is the module itself properly accounting for this and returning an appropriate warning? If not, shouldn't it be?

Reply:
agreed it should return an error. Currently it just returns {} if its not supported. Will add a warning.